src/Main.cpp

```diff
@@ -100,6 +102,11 @@ int main(int argc, char* argv[]) {
   seissol::MPI::mpi.init(argc, argv);
   const int rank = seissol::MPI::mpi.rank();
 
+  if (utils::Env::get<bool>("FLOATING_POINT_EXCEPTION", false)) {
+    logInfo() << "REACHED HERE";
```

Comment:
`logInfo(rank) << "Enabling floating point exception handlers.";` or the likes?

Reply:
Oh, my bad, it seems like I forgot to remove the test statement I used. Putting the right one now, thanks!